Selenium.Axe/Selenium.Axe.Test/AxeRunOptionsTest.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using FluentAssertions;
+
+namespace Selenium.Axe.Test
+{
+    [TestClass]
+    public class AxeRunOptionsTest
+    {
+        private readonly JsonSerializerSettings serializerSettings = new JsonSerializerSettings
```

Comment:
If we're using this, we can also remove all the `[JsonProperty("camelCaseVersionOfProperty")]` attributes in favor of `ContractResolver = new CamelCasePropertyNamesContractResolver()` in the settings here

Reply:
(If you're going to remove the existing use of the serializer settings from the includexcludemanager, I'm okay with going back to all-attributes, too; I just want to be consistent, either using the settings consistently or not-using it consistently :) )

Reply:
this serializer has nothing to do with how this will be serialized in other places. I am using this serializer so that it is easier match against the string that i am comparing with.